core/app/models/spree/null_adjuster.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Spree
+  class NullAdjuster
```

Comment:
Maybe a more specific name would help to understand the purpose of the object.

What about?

```rb
Spree::NullPromotionAdjuster
```

Reply:
I second this question, unless you see a reason to keep it more generic. In that case, can you please help us understand where it is?

Reply:
My observation was that the class doesn't actually do anything promotion-specific, and the interface (`#call`) also doesn't necessarily do anything. I was suspecting (but wrongly) that also tax adjuster might share that interface. 

Whatever: It's completely fine to call it `NullPromotionAdjuster`, and I've amended the PR to reflect that.